### PR TITLE
feat: Set clear expectations about how to prep for the full-stack pairing session

### PR DESCRIPTION
**Pull request by tklarryonline**
Be explicit about:

- The code is supposed to be imperfect
- The candidates don't have to finish the implementation prior to the interview
- Emphasise the "work together" bit

### CODE REVIEW

**Comment by tsommer** [comment]
@tklarryonline - Looks great. I've made some small changes for flow, but otherwise 👍

File: fullstack/README.md [hunk 1]
```diff
@@ -26,4 +29,6 @@ current code.
   - Tests
   - Business logic
   - Design choices
-- It's not perfect, we'd love to hear how you'd improve it
+- We had fun creating it. It's not perfect, we'd love to hear how you'd improve it.
```
Comment (cosmini) [inline fullstack/README.md:32]:
```suggestion
  - We had fun creating it. It's not perfect, we'd love to hear how you'd improve it.
```
Can we please fix this alignment ?

Comment (tklarryonline) [reply fullstack/README.md:32]: That line is aligned with line 27 and was not in the same group with the ones directly above.

**Review by cosmini — APPROVED** [review]

**Review by ishkananrb — APPROVED** [review]

**Comment by tsommer** [comment]
@tklarryonline - let's merge :)

**Pull request merged**